installing_on_google_compute_engine/_topics/installation.md

```diff
@@ -9,35 +9,35 @@ Installing {{ site.data.product.title_short }} consists of the following steps:
 
 3.  Configuring the {{ site.data.product.title_short }} appliance.
 
-After you have completed all the procedures in this guide, you will have
+After you complete all the listed preceding steps, you have
 a working environment on which additional customizations and
 configurations can be performed.
 
 ### Obtaining the Appliance
 
 ### Uploading the Appliance on Google Compute Engine
 
-When the {{ site.data.product.title_short }} Google Compute Engine appliance has finished
-downloading to your system, upload the appliance to Google Compute
-Engine.
+When the {{ site.data.product.title_short }} Google Compute Engine appliance is downloaded to your system, upload the appliance to Google Compute Engine.
 
-To upload the {{ site.data.product.title_short }} Google Compute Engine appliance file you
-will need: \* 44 GB of storage space. \* 12 GB RAM. \* 4 vCPUs.
+To upload the {{ site.data.product.title_short }} Google Compute Engine appliance file, you need to satisfy the following requirements:
+- 44 GB of storage space
+- 12 GB RAM
+- 4 vCPUs.
 
 You can upload the appliance with the following steps:
 
 1.  Log in to the Google Cloud Platform dashboard.
 
-2.  Click **Home** in the top left of the screen.
+2.  Click **Home** located on the left of the screen.
```

Comment:
```suggestion
2.  Click **Home** located in the top left corner of the screen.
```

Reply:
We can't say things like top left bottom right etc. These introduce accessibility violations.